### DISCUSSION TITLE
Error validating transaction: Error running script for input 0 referencing 84f19027c9524ad08e34692be66dd5d1bf2836ffb2bc82adb0d8db7d7b7c9823 at 0: Script was NOT verified successfully..

### DISCUSSION THREAD

**Discussion by justresponse** [trustwallet/wallet-core#3595] [Q&A]
I am an Android developer. When I use the following method to sign, there is no problem, but an error occurs when broadcasting. Below is my signing code:
/**
          * formAddress sending address
          * toAddress receiving address
          * privateKey private key
          *Amount transfer amount
          * Fee handling fee
          * mUtxoList unconsumed utxo collection
          * coinType chain type
          */
         fun organize(fromAddress: String,
                      toAddress: String,
                      privateKey: String,
                      Amount: String,
                      Fee: String,
                      mUtxoList: MutableList<BtcUtxoResultEntity.TxrefsBean>,
                      coinType: CoinType
         ): String{
             val amount = BigDecimal(Amount).multiply(BigDecimal(100000000)).longValueExact()
             val fee = BigDecimal(Fee).multiply(BigDecimal(100000000)).longValueExact()

             val mPrivateKey = PrivateKey(BTCUtil.HEX.decode(privateKey))
             val publicKey = mPrivateKey.getPublicKeySecp256k1(true)
             val pubKeyHash = Hash.ripemd(Hash.sha256(publicKey.data()))

             val script = BitcoinScript.lockScriptForAddress(fromAddress,coinType)

             mUtxoList.sortByDescending { it.value }
             var totalMoney = 0L
             val list = mutableListOf<Bitcoin.UnspentTransaction>()
             for (buff in mUtxoList) {
                 if (totalMoney >= amount+fee) break
                 val txHash = BTCUtil.HEX.decode(buff.tx_hash)
                 txHash.reverse()
                 val outpoint = Bitcoin.OutPoint.newBuilder()
                     .setHash(ByteString.copyFrom(txHash))
                     .setIndex(buff.tx_output_n)
                     .setSequence(Long.MAX_VALUE.toInt())
                     .build()
                 val utxo = Bitcoin.UnspentTransaction.newBuilder()
                     .setAmount(buff.value)
                     .setOutPoint(outpoint)
                     .setScript(ByteString.copyFrom(script.data()))
                     .build()
                 list.add(utxo)
                 totalMoney += buff.value
             }

             val plan = Bitcoin.TransactionPlan.newBuilder().apply {
                 this.addAllUtxos(list)
                 this.fee = fee
                 this.amount = amount
                 this.change = totalMoney-fee-amount
             }
             val mSigningInput = Bitcoin.SigningInput.newBuilder()
                 .setAmount(amount)
                 .setHashType(BitcoinScript.hashTypeForCoin(coinType))
                 .setToAddress(toAddress)
                 .putScripts(toHexString(script.matchPayToScriptHash()),ByteString.copyFrom(BitcoinScript.buildPayToWitnessPubkeyHash(pubKeyHash).data()))
                 .setCoinType(coinType.value())
                 .setChangeAddress(fromAddress)
                 .addPrivateKey(ByteString.copyFrom(mPrivateKey.data()))
                 .addAllUtxo(list)
                 .setByteFee(1)
                 .setPlan(plan)

             val output = AnySigner.sign(mSigningInput.build(), coinType, Bitcoin.SigningOutput.parser())
             val encoded = output.encoded

             Log.e("pino", "ltc_sign_result:${BTCUtil.HEX.encode(encoded.toByteArray())}")
             return BTCUtil.HEX.encode(encoded.toByteArray())
         }
Hope to get help to solve this problem

**Comment by shonsyk** [comment]
Hi,
Have you been able to solve the problem so far?

**Reply by satoshiotomakan** [reply]
Hi @MaxBolotnyi, @justresponse
It's difficult to figure out what's wrong with the transaction you tried to broadcast without the context. Could you please provide `fromAddress`, the list of UTXOs. I hope it will be enough

**Reply by justresponse** [reply]
fromAddress:3F9LMSynCeeBmd2pfoJEM8wm3JV2WJKk9k 
UTXOs:{tx_hash:84f19027c9524ad08e34692be66dd5d1bf2836ffb2bc82adb0d8db7d7b7c9823,value:80000,tx_input_n:-1,tx_output_n:0,script:a9149393d5936cf79b3b480b52f9652c2f2bf04d270087}

**Reply by justresponse** [reply]
Is there a group for easy communication? Such as discord or telegram

**Reply by lamafab** [reply]
So something I noticed in your code:

```kotlin
             val outpoint = Bitcoin.OutPoint.newBuilder()
                 .setH ash(ByteString.copyFrom(txHash))
                 .setIndex(buff.tx_output_n)
                 .setSequence(Long.MAX_VALUE.toInt())
                 .build()
```

You're using the max value of Long (signed, `9223372036854775807`), but it should be the max value of UInt (unsigned, `4294967295`). Also, make sure that `buff.tx_output_n` points to the correct index/utxo of the transaction.
